### DISCUSSION TITLE
Empty Explain/Translation views

### DISCUSSION THREAD

**Discussion by msanchezdev** [community/community#19012] [Copilot Conversations]
Explain and Language translation views render empty and the follwing message appears when trying to reload them by enabling and disabling them on the top right menu of the "Github Copilot Labs" View Container

```
Error loading webview:
  Error: Could not register service workers:
    InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state..
``` 

### Screenshots
![screenshot of empty views](https://user-images.githubusercontent.com/35346082/174697364-09a88f3a-87c2-42ef-b2c2-3ae610f4e3f1.png)
![screenshot of error message](https://user-images.githubusercontent.com/35346082/174697152-1e6892c3-1b2d-4862-8ca9-eb81395e8126.png)

### Notes: 
- Restarted VS Code several times
- Re-logged into Copilot
- Re-installed both Copilot and Copilot Labs

**Comment by melvincwng** [comment]
@msanchezdev, if you believe this to be a bug, please submit a ticket and report it to [GitHub Support](https://support.github.com/contact/bug-report) over here.

They usually respond within a few business days.

If this helps, please kindly 'Mark as answer' so that others know this issue has been addressed.

Thanks!
